composer.json

```diff
@@ -14,11 +14,11 @@
     },
     "require": {
         "php": ">=5.5",
-        "zendframework/zend-eventmanager": "~2.5"
+        "zendframework/zend-eventmanager": "dev-develop as 2.7.0"
```

Comment:
We can actually set this to ^2.6|^3.0, as the changes made in here are compatible with 2.6!

I'll take care of that on merge. That also means we can target this for an immediate 2.6 release, pending any other pull requests.